lib/core/viewer/components/viewer.component.ts

```diff
@@ -672,31 +698,31 @@ export class ViewerComponent implements OnChanges, OnInit, OnDestroy {
         return rendition;
     }
 
-    private async waitRendition(nodeId: string, renditionId: string): Promise<RenditionEntry> {
```

Comment:
all the logic related of the rendition can be moved in a service of this component ?